awx/main/dispatch/pool.py

```diff
@@ -309,6 +337,10 @@ def __init__(self, *args, **kwargs):
         # max workers can't be less than min_workers
         self.max_workers = max(self.min_workers, self.max_workers)
 
+    def debug(self, *args, **kwargs):
+        self.cleanup()
+        return super(AutoscalePool, self).debug(*args, **kwargs)
```

Comment:
The autoscale pool used by the dispatcher cleans up its state to detect orphaned processes before stats are reported.